### DISCUSSION TITLE
[Question] Why default language not work?

### DISCUSSION THREAD

**Discussion by ghost** [gobuffalo/buffalo#2122] [Q&A]
Hi, I have this strange problem:
in the `locales` path i have three language file: 

- all.de.yaml
- all.en.yaml
- all.es.yaml

In the app.go I have:

```
func translations() buffalo.MiddlewareFunc {
	var err error
	if T, err = i18n.New(packr.New("app:locales", "../locales"), "es"); err != nil {
		app.Stop(err)
	}
	return T.Middleware()
}
```
But always the app start with the "de" language. I tryied to change "es" with "en" but it return always "de". Seams that it set the default language getting the first file in alpabetic order. 

What I missing?

**Comment by mardu** [comment]
Your `Accept-Language:` request header probably overrides the default language. Try with some tool that does not sent this header by default for example Postman.
